spark/src/test/java/org/apache/iceberg/spark/data/TestSparkOrcReader.java

```diff
@@ -49,7 +49,7 @@ protected void writeAndValidate(Schema schema) throws IOException {
     }
 
     try (CloseableIterable<InternalRow> reader = ORC.read(Files.localInput(testFile))
-        .schema(schema)
+        .project(schema)
```

Comment:
nit: would it be easier to address this change as a separate PR. Seems un-related to current work

Reply:
OK I split it to another PR